google-cloud-storage/src/main/java/com/google/cloud/storage/Blob.java

```diff
@@ -900,6 +900,7 @@ public Builder toBuilder() {
     return new Builder(this);
   }
 
+  /** Returns true if obj's class, toPb and options are equals */
```

Comment:
Please change to:

```
Returns true if obj instance `<pre>{@code Blob.toPb()}</pre>` metadata representation and `<pre>{@code Blob.options}</pre>` instance of StorageOptions are both equal.
```

Also generate javadocs and take a screenshot of the resulting change.

Reply:
![image](https://user-images.githubusercontent.com/26637492/82197130-287bc700-992d-11ea-9be8-2402718be581.png)

Hi please see the screen shot above

Reply:
Please inline `<pre>{@code Blob.toPb()}</pre>` and `<pre>{@code Blob.options}</pre>`.

Reply:
@frankyn If I put it inline it won't pass the lint test.... mvn com.coveo:fmt-maven-plugin:format auto format like this.

Reply:
Ah, do this instead:

```
Returns true if obj instance {@code Blob.toPb()} metadata representation and {@code Blob.options} instance of StorageOptions are both equal.
```

Reply:
![image](https://user-images.githubusercontent.com/26637492/82675762-dbbf2580-9c77-11ea-9a44-5d2ce9707402.png)